packages/coinstac-ui/test/e2e/test.js

```diff
@@ -0,0 +1,143 @@
+/* eslint func-names: 0, prefer-arrow-callback: 0 */
+const Application = require('spectron').Application;
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+const path = require('path');
+
+const electronPath = path.join(__dirname, '../..', 'node_modules', '.bin', 'electron');
+const appPath = path.join(__dirname, '../..');
+const mocksPath = path.join(__dirname, 'mocks.js');
+
+const EXIST_TIMEOUT = 4000;
+const USER_ID = 'test';
+const PASS = 'test';
+const CONS_NAME = 'e2e-consortium';
+const CONS_DESC = 'e2e-description';
+const PROJECT_NAME = 'e2e-files';
+
+global.before(function () {
```

Comment:
You should be able to drop the `global` here: the engine should resolve `before` to Mocha's global function. Specifying mocha as an [ESLint env config](http://eslint.org/docs/user-guide/configuring#specifying-environments) might help if you’re seeing lint errors.

Also, because `chai.should` and `chai.use` are synchronous you shouldn’t need to wrap them in a `before`.